tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -304,10 +309,16 @@ def get_rocm_test_compilation_infos(
     for schedule in schedules:
         # Skip schedules with an intrinsic which element type does not
         # match the requested one.
-        if lhs_rhs_type.value.upper() not in schedule.intrinsic:
+        # Search for the lhs_rhs type in the first part of intrinsic
+        # e.g., MFMA_F32_16x16x4_F32 -> MFMA_F32
+        if lhs_rhs_type.value.upper() not in schedule.intrinsic[:8]:
```

Comment:
Would this work with `MFMA_F8E4M3FNUZ` intrinsic?

Reply:
The current logic won't. I've updated the script to find the right pattern. PTAL.